addons/test_discuss_full/tests/test_performance.py

```diff
@@ -205,6 +205,16 @@ def _get_init_store_data_result(self):
                     "user_id": {"id": self.users[0].id},
                     "voice_active_duration": 0,
                     "volumes": [("ADD", [])],
+                    "google_calendar_cal_id": False,
+                    "google_calendar_rtoken": False,
+                    "google_calendar_sync_token": False,
+                    "google_calendar_token": False,
+                    "google_calendar_token_validity": False,
+                    "google_synchronization_stopped": False,
+                    "microsoft_calendar_sync_token": False,
+                    "microsoft_last_sync_date": False,
+                    "microsoft_synchronization_stopped": False,
```

Comment:
The two specific modules should be added to dependencies of this test module then.